### PR TITLE
NAS-102713 add help text for new SNMPv3 trap fields.

### PR DESCRIPTION
**Pull request by a-stjohn**
Will wait until this text is approved before making userguide changes.

### CODE REVIEW

File: src/app/helptext/system/alert-service.ts [hunk 1]
```diff
@@ -151,25 +151,52 @@ export default {
...
+    SNMPTrap_v3_privprotocol_placeholder: T('Encryption Protocol'),
+    SNMPTrap_v3_privprotocol_tooltip: T('Encryption protocol \
+ used to encrypt messages sent on behalf of the specified \
+ <i>Username</i>. Supported encryption protocols are shown at \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmNoPrivProtocol" \
+ target="_blank">http://snmplabs.com/</a>'),
 
     SNMPTrap_community_placeholder: T('SNMP Community'),
-    SNMPTrap_community_tooltip: T(''),
+    SNMPTrap_community_tooltip: T('Enter the network community string. \
```
Comment (Mrt134) [inline src/app/helptext/system/alert-service.ts]:
```suggestion
    SNMPTrap_community_tooltip: T('Network community string. \
```

File: src/app/helptext/system/alert-service.ts [hunk 2]
```diff
@@ -151,25 +151,52 @@ export default {
...
-
-    SNMPTrap_v3_privprotocol_placeholder: T('Encryption protocol'),
-    SNMPTrap_v3_privprotocol_tooltip: T(''),
+    SNMPTrap_v3_username_tooltip: T('Username of the SNMP \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#user-based" \
+ target="_blank">User-based Security Model (USM)</a> user.'),
+
+    SNMPTrap_v3_authkey_placeholder: T('Secret Authentication Key'),
+    SNMPTrap_v3_authkey_tooltip: T('Initial secret authentication key. \
+ If an authentication key is not set, no <i>Authentication Protocol</i> is \
```
Comment (Mrt134) [inline src/app/helptext/system/alert-service.ts]: Is "if" or "when" the clearer term here?

File: src/app/helptext/system/alert-service.ts [hunk 3]
```diff
@@ -151,25 +151,52 @@ export default {
...
+ <i>Username</i>. Supported authentication protocols are shown at \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmNoAuthProtocol" \
+ target="_blank">http://snmplabs.com/</a>.'),
+
+    SNMPTrap_v3_privprotocol_placeholder: T('Encryption Protocol'),
+    SNMPTrap_v3_privprotocol_tooltip: T('Encryption protocol \
+ used to encrypt messages sent on behalf of the specified \
+ <i>Username</i>. Supported encryption protocols are shown at \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmNoPrivProtocol" \
+ target="_blank">http://snmplabs.com/</a>'),
```
Comment (Mrt134) [inline src/app/helptext/system/alert-service.ts]: Feels a little odd to use a link for link text when the actual link isn't going to the same place.

File: src/app/helptext/system/alert-service.ts (hunk 2 above)
Comment (a-stjohn) [inline src/app/helptext/system/alert-service.ts]:
```suggestion
 When an authentication key is not set, no <i>Authentication Protocol</i> is \
```

File: src/app/helptext/system/alert-service.ts [hunk 4]
```diff
@@ -151,25 +151,52 @@ export default {
...
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmDESPrivProtocol" \
+ target="_blank">DES</a> is used as the default.<br> \
+ Must be at least 8 and at most 32 octets long.'),
+
+    SNMPTrap_v3_authprotocol_placeholder: T('Authentication Protocol'),
+    SNMPTrap_v3_authprotocol_tooltip: T('Authentication protocol \
+ used to authenticate messages sent on behalf of the specified \
+ <i>Username</i>. Here is a \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmNoAuthProtocol" \
+ target="_blank">list of supported authentication protocols</a>.'),
```
Comment (wblock) [inline src/app/helptext/system/alert-service.ts]: Make the words _Authentication protocol_ in the first sentence have this link, and then the second sentence is not needed.

File: src/app/helptext/system/alert-service.ts [hunk 5]
```diff
@@ -151,25 +151,52 @@ export default {
...
+
+    SNMPTrap_v3_privprotocol_placeholder: T('Encryption Protocol'),
+    SNMPTrap_v3_privprotocol_tooltip: T('Encryption protocol \
+ used to encrypt messages sent on behalf of the specified \
+ <i>Username</i>. Supported encryption protocols are shown at \
+ <a href="http://snmplabs.com/pysnmp/docs/api-reference.html#pysnmp.hlapi.usmNoPrivProtocol" \
+ target="_blank">http://snmplabs.com/</a>'),
```
Comment (wblock) [inline src/app/helptext/system/alert-service.ts]: Can do the same thing here as above: make _Encryption protocol_ in the first sentence the link from the second, then remove the second.

**Pull request merged**